### DISCUSSION TITLE
USB Disk disconnectes, all Playlist empty

### DISCUSSION THREAD

**Discussion by Mixnetwork** [AzuraCast/AzuraCast#5834] [General]
I am running latest Azuracast version v0.17.4 Stable • Docker • PHP 8.1 

Installed with recommend setup for Ubuntu with Docker.

In Ubuntu I mounted my USB Disk with all the Mediafiles in thee Emby Station directory.
While setting up all everything works so far.
I made Playlist for the whole week and programmed them in the planning tool. After rebooting, everything still ok.
Somehow after one day (Station was running permenantely) the USB drive was not connected anymore. I do not know why, becausse Power was there and nothing else happened.
So I reconnected again with the Mount command in the same directory.

Files where all available again, but now the schock, all playlists where empty, no songs in it.
When I look  at the songs they where not yet scanned anymore, and he was scanning again same songs.

Also a  reboot did not help, I must do all over again. 

IS there a solution that he does not delete the Info of the media files? Perhaps add them to a database? Because this is a horrible scenario and a lot of work, when I must do all over again just because of that the USB Drive was disconnected

**Comment by xewl** [comment]
Sort the files in folders, assign the folders to the Playlist, instead of the files directly. They'll pop back up when rescanned.

**Reply by Mixnetwork** [reply]
OK I will try this. thx for response
